hummingbird/ml/operator_converters/onnx/sv.py

```diff
@@ -0,0 +1,96 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Converters for ONNX-ML SV models.
+"""
+
+import numpy as np
+from onnxconverter_common.registration import register_converter
+
+from .._sv_implementations import SVC
+
+
+def convert_onnx_svm_classifier_model(operator, device, extra_config):
+    """
+    Converter for `ai.onnx.ml.SVMClassifier`
+
+    Args:
+        operator: An operator wrapping a `ai.onnx.ml.SVMClassifier` model
+        device: String defining the type of device the converted operator should be run on
+        extra_config: Extra configuration used to select the best conversion strategy
+
+    Returns:
+        A PyTorch model
+    """
+    operator_orig = operator
```

Comment:
Can we avoid this and just call operator.raw_operator (instead of operator) when required later in the code (apparently only in line 38)